lib/OpenQA/Script/CloneJob.pm

```diff
@@ -193,8 +193,10 @@ sub create_url_handler ($options) {
         api => $local_url->host,
         apikey => $options->{apikey},
         apisecret => $options->{apisecret});
-    die "API key/secret for '$options->{host}' missing. Checkout '$0 --help' for the config file syntax/lookup.\n"
-      unless $local->apikey && $local->apisecret;
+    if (!$options->{'export-command'}) {
```

Comment:
`unless ($options->{'export-command'})` looks more suitable here. wdyt?

Reply:
I thought it might become less readable since there's already an unless here... no strong opinion, though

Reply:
https://github.com/os-autoinst/openQA/pull/5871#discussion_r1724980157